spaces/S000145/properties/P000138.md

```diff
@@ -0,0 +1,10 @@
+---
+space: S000145
+property: P000138
+value: false
+---
+
+Let $A = \{a_0, a_1, \dots\}$ be any infinite closed subset of $X$, and take arbitrary $p \in X \setminus A$.
+Then for each $S \subseteq \mathbb N$, the function \[ f_S(x) = \begin{cases} p & x = a_i, i \in S, \\ x & \text{otherwise}, \end{cases} \] is continuous since the preimage of any open subset $U$ is of the form $U \setminus Q$ or $\left( U \cup \{p\} \right) \setminus Q$ where $Q \subseteq A$ is closed.
+
+These $f_S$ induced continuum many continuous function from $X$ to itself.
```

Comment:
```suggestion
Let $A$ be an infinite closed proper subset of $X$, and take an arbitrary $p \in X \setminus A$.
For each $S \subseteq A$, the function $f_S:X\to X$ defined by
$$f_S(x) = \begin{cases} p & \text{if }x\in S, \\ x & \text{otherwise}, \end{cases}$$
is continuous since for every nonempty open set $U$, its inverse image $f^{-1}(U)$ contains $U\setminus A$, which is open and nonempty; hence $f^{-1}(U)$ is open.

And there are continuum many such functions $f_S$.
```